(dp1
S'output'
p2
S"<type 'exceptions.RuntimeError'> Table: missing required field: metadata"
p3
sS'layer'
p4
S'/Users/ecain/Projects/SOTA/web2py/applications/SOTA/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x114e57b30, file "/Users/ecain/Projects/SOTA/web2py/applications/SOTA/controllers/default.py:index", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/restricted.py", line 216, in restricted\n    exec(ccode, environment)\n  File "/Users/ecain/Projects/SOTA/web2py/applications/SOTA/controllers/default.py:index", line 156, in <module>\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/globals.py", line 405, in <lambda>\n    self._caller = lambda f: f()\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/tools.py", line 4321, in f\n    return action(*a, **b)\n  File "/Users/ecain/Projects/SOTA/web2py/applications/SOTA/controllers/default.py:index", line 29, in index\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/tools.py", line 4321, in f\n    return action(*a, **b)\n  File "/Users/ecain/Projects/SOTA/web2py/applications/SOTA/controllers/default.py:index", line 88, in update_form\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/html.py", line 2298, in process\n    self.validate(**kwargs)\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/html.py", line 2236, in validate\n    if self.accepts(**kwargs):\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/sqlhtml.py", line 1857, in accepts\n    self.vars.id = self.table.insert(**fields)\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/packages/dal/pydal/objects.py", line 741, in insert\n    ret = self._db._adapter.insert(self, self._listify(fields))\n  File "/Users/ecain/Projects/SOTA/web2py/gluon/packages/dal/pydal/objects.py", line 680, in _listify\n    \'Table: missing required field: %s\' % name)\nRuntimeError: Table: missing required field: metadata\n'
p11
s.